### ISSUE TITLE
Userscript failing with a CSP violation error

### ISSUE THREAD

**Issue by chasingmaxwell** [Tampermonkey/tampermonkey#328]
```
Refused to load the font 'data:font/woff;base64,d09GRgABAAAAAIw4ABEAAAABQcAAAQABAAAAAAAAAAAAAAAAAAAAA…hgZiCKVViwAiVhsAFFYyNisAIjRLMJCgMCK7MLEAMCK7MRFgMCK1myBCgGRVJEswsQBAIrAA==' because it violates the following Content Security Policy directive: "font-src assets-cdn.github.com".
```

I'm receiving this error many times in the console when trying to use the [GitHub Dark](https://github.com/StylishThemes/GitHub-Dark-Script) userscript on chrome. I commented on a [related issue](https://github.com/StylishThemes/GitHub-Dark-Script/issues/13#issuecomment-256921477) in their queue and they directed me here. I have the "Add Tampermonkey to the site's content security policy (CSP) if there is one" setting configured to "Yes". The userscript is having no effect on the page at all so I'm hoping this is the cause as I've got very little else to go on.

Tampermonkey version: v4.1.10
Chrome version: 54.0.2840.71 (64-bit)
OS: Ubuntu 15.10

**Comment by chasingmaxwell** [comment]
Update: the issue of the userscript not functioning was a separate issue. I do, however, still see the above error in the console.

**Comment by derjanb** [comment]
@chasingmaxwell I can not reproduce this. Is there something that I need to do in order to make the font load. 
It also seems that the styles are injected via a normal style tag and not GM_addStyle (which is used only once to inject some static styles). Using GM_addStyle should prevent CSP issues.

**Comment by Mottie** [comment]
> the styles are injected via a normal style tag and not GM_addStyle

@derjanb We didn't use `GM_addStyle` because we are not able to control it the style once it has been applied, e.g. updating, removing, enabling or disabling the style. Maybe if there was some easy way to get the id of the added style element... but then again every userscript engine may do something a little different.

**Comment by derjanb** [comment]
@Mottie ```GM_addStyle``` returns the injected DOM element. I'm not sure if it can be modified, but removal and re-adding a new one should work.

**Comment by tophf** [comment]
Re-assigning its `textContent` property should work, presumably.

**Comment by Mottie** [comment]
> `GM_addStyle` returns the injected DOM element.

Oh LOL, I never tried that! I don't think it's documented anywhere.

**Comment by derjanb** [comment]
Documented. :)
